superset/charts/schemas.py

```diff
@@ -156,7 +156,6 @@ class ChartEntityResponseSchema(Schema):
     slice_name = fields.String(description=slice_name_description)
     cache_timeout = fields.Integer(description=cache_timeout_description)
     changed_on = fields.String(description=changed_on_description)
-    modified = fields.String()
```

Comment:
By catch, this field returns invalid strings:
<img width="808" alt="Xnip2022-05-05_14-16-49" src="https://user-images.githubusercontent.com/335541/167033095-efae0202-3721-4918-b66a-726eb1177d74.png">